### ISSUE TITLE
Is drag and drop available yet?

### ISSUE THREAD

**Issue by dbreen1** [thellmund/Android-Week-View#263]
Hi, 

Thanks for the library. 

I see that you merged the changes for drag and drop a few months ago. I pulled the latest version (5.3.2) of the library but the changes don't appear to be there.

Is the feature still in progress?

Cheers

**Comment by redlightmikey** [comment]
@thellmund Just looking at this. Any chance of add this? Would love to add this!

**Comment by rezazarchi** [comment]
Yes, it's already available. And I just wonder how to disable this feature? @thellmund

**Comment by jom16antonio** [comment]
> Yes, it's already available. And I just wonder how to disable this feature? @thellmund



> Drag and drop is available in v5.3.2. I also wonder how to disable this feature. Also how to capture the dragdrop changes when you are using simpleadaptor?

To disable drag and drop, return true in the onEventLongClick()
`override fun onEventLongClick(data: DayItemDto, bounds: RectF): Boolean {
     return true
}
`

**Comment by eliomaroun** [comment]
@jom16antonio Return type is 'Boolean', which is not a subtype of overridden
I am getting this error when applying what you did. Any idea how to resolve it.
I am using WeekViewPagingAdapterJsr310 
using the latest release of weekview in android 13 
Thanks!

**Comment by JSroka745** [comment]
I also use 5.3.2 version and not see drag and drop feature. how to enable it?
